elm.json

```diff
@@ -13,9 +13,11 @@
             "elm/random": "1.0.0",
             "elm/url": "1.0.0",
             "elm-community/random-extra": "3.2.0",
-            "fapian/elm-html-aria": "1.4.0"
+            "fapian/elm-html-aria": "1.4.0",
+            "pablen/toasty": "1.2.0"
```

Comment:
What does this do?

Reply:
Not a lot. I might end up getting rid of it.